liveness-alert.tf

```diff
@@ -10,7 +10,7 @@ module "bulk-scan-processor-liveness-alert" {
   app_insights_query = <<EOF
 requests
 | where name == "GET /health" and resultCode != "200"
-| where url contains "processor"
+| where url contains "bulk-scan-processor"
```

Comment:
there is now also a service called 'bulk-scan-payment-processor', so just `contains "processor"` would pick up both